src/tests/functional/plugin/conformance/test_runner/api_conformance_runner/src/ov_plugin/core_integration.cpp

```diff
@@ -15,17 +17,17 @@ namespace {
 // OV Class Common tests with <pluginName, deviceName params>
 //
 
-INSTANTIATE_TEST_SUITE_P(ov_plugin_mandatory,
-                         OVClassModelTestP,
-                         ::testing::Values(ov::test::utils::target_device));
+INSTANTIATE_TEST_SUITE_P(ov_plugin_mandatory, OVClassModelTestP, ::testing::Values(ov::test::utils::target_device));
 
 INSTANTIATE_TEST_SUITE_P(ov_plugin,
                          OVClassModelOptionalTestP,
                          ::testing::Values(ov::test::utils::target_device));
 
 // OV Class Query network
 
-INSTANTIATE_TEST_SUITE_P(ov_plugin_mandatory,
+static std::string (*NULL_getTestCaseName)(const testing::TestParamInfo<std::string>& info) = NULL;
+INSTANTIATE_TEST_SUITE_P(ov_plugin,
                          OVClassQueryModelTest,
-                         ::testing::Values(ov::test::utils::target_device));
+                         ::testing::Values(ov::test::utils::target_device),
+                         MARK_MANDATORY_API_FOR_HW_DEVICE(NULL_getTestCaseName));
```

Comment:
Could you please explain the technical details of the changes. I why you create NULLTestCaseName for sw plugin... Is there any other approaches? I asked you because it is not so clear...

Reply:
The reason is to support 2 types test cases with unify MACRO:
Some API test cases have implemented getTestCaseName() 
For example OVHoldersTestOnImportedNetwork::getTestCaseName
Some are not. For example OVGetMetricPropsTest

Another option, define 2 MACROs:
MARK_MANDATORY_API_FOR_HW_DEVICE_0 without parameter.
MARK_MANDATORY_API_FOR_HW_DEVICE_1 with parameter

Reply:
Is it possible to rename as `WITH_PARAM` and `WITHOUT_PARAM`

In general, LGTM